xcm/src/v3/mod.rs

```diff
@@ -678,6 +677,7 @@ pub enum Instruction<Call> {
 	///
 	/// Errors:
 	/// - `ExpectationFalse`: In case any of the expectations are broken.
+	#[weight_args(index: &u32)]
```

Comment:
And why modify them here?

Reply:
The weight function for this instruction only takes in an `index: &u32` and not all of the fields in the variant, so it's modified.

Reply:
same as above